src/app/data-model-provider/MetadataTypes.h

```diff
@@ -75,6 +87,11 @@ struct ClusterInfo
     ClusterInfo(DataVersion version) : dataVersion(version) {}
 };
 
+// constants used to narrow binary expressions
+constexpr uint8_t kPrivilegeBits   = 5;
+constexpr uint8_t kAttrQualityBits = 7;
+constexpr uint8_t kCmmdQualityBits = 3;
```

Comment:
```suggestion
constexpr uint8_t kCmdQualityBits = 3;
```

I think the typical mnemonic is `cmd`. Maybe consider even not trying to use short name and spell out Attribute and Command.

Reply:
Fixed.